themes/classic/templates/_partials/form-fields.tpl

```diff
@@ -41,7 +41,15 @@
       {if $field.type === 'select'}
 
         {block name='form_field_item_select'}
-          <select class="form-control form-control-select" name="{$field.name}" {if $field.required}required{/if}>
+          <select 
+            {if empty($field.attributes)} 
+                class="form-control form-control-select" name="{$field.name}" {if $field.required}required{/if} 
```

Comment:
Just notice that you always need to set all attributes :thinking: 
Why not having default attributes per field type?
Then be able to overwrite then with `setAttributes`?